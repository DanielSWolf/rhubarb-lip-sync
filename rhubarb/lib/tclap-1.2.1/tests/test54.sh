#!/bin/sh

# success
../examples/test8 --help  > tmp.out 2>&1

if cmp -s tmp.out $srcdir/test54.out; then
	exit 0
else
	exit 1
fi

