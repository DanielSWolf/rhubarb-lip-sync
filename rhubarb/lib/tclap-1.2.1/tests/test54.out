
USAGE: 

   ../examples/test8  [-f=<float>] ...  [-i=<int>] ...  -s=<string> [-B]
                      [--] [--version] [-h] <string> <string>
                      <fileNameString> ...


Where: 

   -f=<float>,  --floatTest=<float>  (accepted multiple times)
     multi float test

   -i=<int>,  --intTest=<int>  (accepted multiple times)
     multi int test

   -s=<string>,  --stringTest=<string>
     (required)  string test

   -B,  --existTestB
     exist Test B

   --,  --ignore_rest
     Ignores the rest of the labeled arguments following this flag.

   --version
     Displays version information and exits.

   -h,  --help
     Displays usage information and exits.

   <string>
     (required)  unlabeled test one

   <string>
     (required)  unlabeled test two

   <fileNameString>  (accepted multiple times)
     file names


   this is a message

